### ISSUE TITLE
Usability nits in `edgedb instance create/destroy`

### ISSUE THREAD

**Issue by elprans** [edgedb/edgedb-cli#593]
A few minor issues with `edgedb instance create` and `destroy`.

1. There is a pointless "is already installed" message:

```
$ edgedb instance create foo
Version 1.0-rc.3+147eb62 is already installed
```

2. Instance create should suggest how to connect to the new instance (`edgedb -I foo`):

```
$ edgedb instance create foo
...
Instance foo is up and running.
```

3. `edgedb instance destroy` silently destroys an instance without confirmation and doesn't output anything:

```
$ edgedb instance destroy foo
$
```

I think we must ask for confirmation by default and print "Instance foo has been destroyed" once done.

**Comment by tailhook** [comment]
I don't see (1) as pointless. It at least notes that there was no install process and that the error that follows this message is unrelated to installation.

We can change the actual message printed in this specific case as opposed to `instance install` but I fail to create a better message for all the cases (create, revert, upgrade, project init) that doesn't repeat other messages but still isn't "is already installed" or very similar to that.

**Comment by tailhook** [comment]
Fixed in #633 except (1).
